egs/wsj/s5/steps/data/augment_data_dir.py

```diff
@@ -175,13 +176,15 @@ def main():
         new_wav = AugmentWav(utt, wav, dur, fg_snrs, bg_snrs, fg_noise_utts,
             bg_noise_utts, noise_wavs, noise_reco2dur, args.fg_interval,
             num_bg_noises)
-        new_utt = utt + "-" + args.utt_suffix
+        new_utt = utt + "-" + args.utt_suffix 
```

Comment:
Sorry, can you also remove the trailing whitespace? You can set up your vimrc to do this for you automatically: https://stackoverflow.com/questions/356126/how-can-you-automatically-remove-trailing-whitespace-in-vim.